src/Control/Effect/Writer.hs

```diff
@@ -56,6 +72,8 @@ listens f m = send (Listen m (curry pure . f))
 --
 --   prop> run (execWriter (censor (applyFun f) (tell (Sum a)))) === applyFun f (Sum a)
 --   prop> run (execWriter (tell (Sum a) *> censor (applyFun f) (tell (Sum b)) *> tell (Sum c))) === (Sum a <> applyFun f (Sum b) <> Sum c)
+--
+-- @since 1.0.0.0
```

Comment:
So, about these. Most of them are incorrect since the functions in question have been around for ages—some of them over a year 😬

What do we want to do about that?

Reply:
Oh, you’re right. I was thinking it was technically inaccurate, since we moved a lot of this stuff around, but this module wasn’t one of the things we moved around. Will fix.